azkaban-exec-server/src/main/java/azkaban/execapp/AzkabanExecServerModule.java

```diff
@@ -30,11 +37,49 @@
  */
 public class AzkabanExecServerModule extends AbstractModule {
 
+  private static final Logger logger = Logger.getLogger(AzkabanExecServerModule.class);
+  private static final String EVENT_REPORTER_CLASS_PARAM = "event.reporter.class";
```

Comment:
Why isn't it prefixed with event.reporting like other configs?

Reply:
Reporter(noun) does reporting(verb). The noun form felt more apt to me especially for the config defining the class. I agree with you for the consistency part. I can keep it all as reporting. Let me know.

Reply:
The convention is to prefix all azkaban config keys with azkaban.
Reporting can be interpreted as a noun too such as financial reporting, right?
Something like
azkaban.event.reporting....?

Reply:
done.